packages/tasks/src/tasks/object-detection/data.ts

```diff
@@ -47,12 +47,12 @@ const taskData: TaskDataCustom = {
 			id: "facebook/detr-resnet-50",
 		},
 		{
-			description: "Real-time and accurate object detection model.",
-			id: "jameslahm/yolov10x",
+			description: "Accurate object detection model.",
+			id: "IDEA-Research/dab-detr-resnet-50",
 		},
 		{
-			description: "Fast and accurate object detection model trained on COCO and Object365 datasets.",
-			id: "PekingU/rtdetr_r18vd_coco_o365",
+			description: "Fast and accurate open-source object detection model.",
```

Comment:
dab-detr-resnet-50 is open source too, isn't it?

Reply:
I actually added it to make a bit of a distinction from waldo, but just noticed rest is open source too. I removed open source indication now